Sources/FocusViewType.swift

```diff
@@ -0,0 +1,17 @@
+//
+//  FocusViewType.swift
+//  BarcodeScanner
+//
+//  Created by Carl Brusell on 12/07/17.
+//  Copyright © 2017 Hyper Interaktiv AS. All rights reserved.
```

Comment:
Could you remove copyright comments? There is LICENSE.md file where everything is covered. You can leave the line with your name if you really want 🙂